src/Utils.php

```diff
@@ -13,6 +13,24 @@ class Utils
     ];
 
     /**
+     * Returns true if the value is truthy
+     *
+     * @param mixed $value Value to check
+     *
+     * @return bool
+     */
+    public static function isTruthy($value)
+    {
+        if (!$value) {
+            return $value === 0 || $value === '0';
+        } elseif ($value instanceof \stdClass) {
+            return !empty(get_object_vars($value));
```

Comment:
Is `!empty` any more performant than a `(bool)` cast? The latter would also work in 5.4.